docs/user_guide/7_restore_backup.md

```diff
@@ -0,0 +1,147 @@
+# Shadow Backup USB stick
+
+You can only follow these steps if you have been provided with a Shadow Backup USB stick. It has a 500 MB Clonezilla partition and a several GB for Clonezilla disk images. These can be used to restore any Shadow-provided laptop(s) and/or NUC(s) to their exact configuration (it's a full disk image restore) after the delivery was tested in Shadow's offices but before it was shipped to the customer. Doing this means losing all the data on the laptop(s) and/or NUC(s) since the original shipment of the equipment from Shadow.
+ 
```

Comment:
so please back up your data to an external device. I would make this bold to make sure they understand that.

Reply:
Fixed in the last commit